test/extensions/common/wasm/wasm_test.cc

```diff
@@ -708,12 +707,10 @@ TEST_P(WasmCommonTest, VmCache) {
   wasm_handle.reset();
   wasm_handle2.reset();
 
-  auto wasm = wasm_handle_local->wasm().get();
+  auto wasm = wasm_handle_local->wasm();
```

Comment:
Note: This test (and 2 more below) were previously testing that `wasm` raw pointer outlived `wasm_handle_local` which, once freed, gets refcount to zero and triggers destructor for the Wasm instance, so I'm not sure what it was supposed to test and for what purpose.